src/components/TodoHeader.tsx

```diff
@@ -0,0 +1,62 @@
+import { FC } from 'react';
+import { CreatedTodo, Todo } from '../types/Todo';
+
+interface Props {
+  title: string;
+  setTitle: (value: string) => void;
+  handleSubmit: (data: CreatedTodo) => Promise<Todo | void>;
+  userId: number;
+  todos: Todo[];
+  setTodos: (todos: Todo[]) => void;
+}
+
+export const TodoHeader: FC<Props> = ({
+  title,
+  setTitle,
+  handleSubmit,
+  userId,
+  todos,
+  setTodos,
+}) => {
+  const isAllTodosComleted = todos.every(todo => todo.completed === true);
+
+  const handleToggleAll = () => {
+    const updatedTodos = todos.map(todo => ({
+      ...todo,
+      completed: !isAllTodosComleted,
+    }));
+
+    setTodos(updatedTodos);
+  };
+
+  return (
+    <header className="todoapp__header">
+      {/* Toggle all todos */}
+      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
```

Comment:
let's fix this eslint issue:smiley:

try to give `aria-label` to button, i think it can help